notebooks/benchmark/Default Mask Benchmark.ipynb

```diff
@@ -0,0 +1,283 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np \n",
+    "import pandas as pd \n",
+    "import xgboost\n",
+    "import shap \n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from shap.benchmark import SequentialPerturbation\n",
+    "from shap.benchmark import get_benchmark, trend_plot, compare_plot"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "metrics = {'sort_order': ['positive', 'negative'], 'perturbation': ['keep']}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_generator = lambda: xgboost.XGBRegressor(n_estimators=100, subsample=0.3)\n",
```

Comment:
instead of lambda just create the model:
untrained_model = xgboost.XGBRegressor(n_estimators=100, subsample=0.3)

Reply:
Will fix that. The old API requires a model generator, which is no longer present with the new framework. I didn't catch that while I was writing the code. Thanks for pointing out!